diagnostic.md

```diff
@@ -33,7 +39,9 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
+After line 35 each value is 3 since the code reads from right to left
+After line 36 y is 10 and x is 4. Since x wasn't stated with it's new value of 3
+it reverts back to 4.
```

Comment:
the final values would actually be x = 3 and y = 10. Reason being, you are reassigning x to the y value from directly above it and THEN y is reassigned.